### PR TITLE
Get rid of rekey functionality (more harm than good)

### LINKED ISSUE #269: Check if sth is going wrong with rekeyed BGP updates on detection's side
**Describe the bug**
Without detection dying, I sometimes see BGP updates not handled (will try to formalize the replication setup; I am doing experiments with PEERING).

**Affected Component(s)**
- [X] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

**To Reproduce**

**Expected behavior**
All BGP updates should be eventually handled.

### PR DESCRIPTION
**Pull request by vkotronis**
### Description of PR

What component(s) does this PR affect?

- [X] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #269

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [X] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by slowr** [comment]
:running: **Benchmark Results** :running:
- BGP Updates (updates/sec): **700**
- Hijacks Detected (hijacks/sec): **28**

File: monitor/core/taps/betabmp.py [hunk 1]
```diff
@@ -57,94 +57,102 @@ def run_bgpstream_beta_bmp(prefixes_file=None):
...
-                                for msg in msgs:
-                                    key_generator(msg)
-                                    log.debug(msg)
-                                    producer.publish(
-                                        msg,
-                                        exchange=exchange,
-                                        routing_key="update",
-                                        serializer="json",
+    # even if connection is closed, retry
+    while True:
```
Comment (vkotronis) [inline monitor/core/taps/betabmp.py]: these 2 lines were practically added

File: monitor/core/taps/betabmp.py [hunk 2]
```diff
@@ -57,94 +57,102 @@ def run_bgpstream_beta_bmp(prefixes_file=None):
...
+                                    else:
+                                        log.warning(
+                                            "Invalid format message: {}".format(msg)
+                                        )
+                                    break
+                        try:
+                            elem = rec.get_next_elem()
+                        except Exception:
+                            continue
+        except Exception:
```
Comment (vkotronis) [inline monitor/core/taps/betabmp.py]: these 2 lines were practically added

File: monitor/core/taps/bgpstreamlive.py [hunk 3]
```diff
@@ -65,100 +65,102 @@ def run_bgpstream(prefixes_file=None, projects=[], start=0, end=0):
...
-                                for msg in msgs:
-                                    key_generator(msg)
-                                    log.debug(msg)
-                                    producer.publish(
-                                        msg,
-                                        exchange=exchange,
-                                        routing_key="update",
-                                        serializer="json",
+    # even if connection is closed, retry
+    while True:
```
Comment (vkotronis) [inline monitor/core/taps/bgpstreamlive.py]: these 2 lines were practically added

File: monitor/core/taps/bgpstreamlive.py [hunk 4]
```diff
@@ -65,100 +65,102 @@ def run_bgpstream(prefixes_file=None, projects=[], start=0, end=0):
...
+                                    else:
+                                        log.warning(
+                                            "Invalid format message: {}".format(msg)
+                                        )
+                                    break
+                        try:
+                            elem = rec.get_next_elem()
+                        except Exception:
+                            continue
+        except Exception:
```
Comment (vkotronis) [inline monitor/core/taps/bgpstreamlive.py]: these 2 lines were practically added

**Comment by slowr** [comment]
:running: **Benchmark Results** :running:
- BGP Updates (updates/sec): **651**
- Hijacks Detected (hijacks/sec): **28**

**Comment by slowr** [comment]
:running: **Benchmark Results** :running:
- BGP Updates (updates/sec): **656**
- Hijacks Detected (hijacks/sec): **21**

**Pull request merged**